lib/ControlledVocab/tests/mountComponent.js

```diff
@@ -105,30 +105,61 @@ export default async function (editable, server) {
   const onEditHandler = sinon.spy();
   const onDeleteHandler = sinon.spy();
 
-  mount(
-    <ConnectedComponent
-      baseUrl="location-units/institutions"
-      records="locinsts"
-      label="Institutions"
-      labelSingular="Institution"
-      objectLabel="Location"
-      visibleFields={['name', 'code', 'source']}
-      columnMapping={{
-        name: 'name',
-        code: 'code',
-        source: 'source',
-      }}
-      readOnlyFields={['source']}
-      itemTemplate={{ source: 'local' }}
-      hiddenFields={['description', 'numberOfObjects']}
-      nameKey="name"
-      // columnWidths={{ 'name': 300, 'code': 50 }}
-      actionSuppressor={{ edit: onEditHandler, delete: onDeleteHandler }}
-      id="institutions"
-      sortby="name"
-      editable={editable}
-    />
-  );
+  labelSingular ?
```

Comment:
why not to prepare specific props based on condition instead of putting whole component mount under condition?

Reply:
Updated to handle through props.